oggm_3dviz/tools/utils.py

```diff
@@ -310,10 +311,81 @@ def init_side_by_side_plotter(initial_time_step=0):
         plotter.close()
 
 
-def get_camera_position_per_frame(start_point, end_point, nr_frames):
+def get_camera_position_per_frame(x_coordinates, y_coordinates, z_elevation, start_point, end_point, nr_frames):
+    """
+    Generates a list of camera positions to move linearly from start_point to end_point over a specified number of frames.
+
+    Parameters:
+    - x_coordinates, y_coordinates: Coordinates that define the range of the scene, used for scaling.
+    - z_elevation: Elevation data of the scene, used to determine vertical range.
+    - start_point, end_point: Starting and ending coordinates of the camera path.
+    - nr_frames: Total number of frames in the animation.
+
+    Returns:
+    - List of tuples representing the camera position for each frame.
+    """
+    # Calculate the range for each axis to scale the camera position
+    x_range = abs(x_coordinates[-1] - x_coordinates[0])
```

Comment:
as you use this scaling multiple times, it maybe is a good idea to add this as a method to `Glacier3DViz`. Something like `get_absolute_x(x_normalized)`.

Reply:
I implemented  get_absolute_coordinates(), because I scale x and y with the longest axis. so that circles don't become oval if the map is not a square.